SellerMarket/locustfile_new.py

```
@@ -125,9 +125,12 @@ def decode_captcha(im: str) -> str:
     
     try:
         response = requests.post(url, headers=headers, json=data)
-        result = response.text
+        result = response.text.strip()
+        # Remove quotes if the response includes them
+        if result.startswith('"') and result.endswith('"'):
+            result = result[1:-1]
         logger.debug(f"Captcha decoded: {result}")
-        return "".join(result)
+        return result
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)